### PR TITLE
caption is changed

### PR DESCRIPTION
**Pull request by skrishn**
hi!

### CODE REVIEW

**Review by jimmieego — APPROVED** [review]
Nice work!

File: _posts/0000-01-06-skrishn.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![spongebob-rainbow](https://cloud.githubusercontent.com/assets/16547949/25401179/038f12ee-29c3-11e7-8362-f92422b7da35.jpg)
 {: .center}
 
-Caption here
+G-i-t-hub FUN!
```
Comment (skrishn) [inline _posts/0000-01-06-skrishn.md:9]: thoughts?

Comment (skrishn) [reply _posts/0000-01-06-skrishn.md:9]: 🐖

Comment (skrishn) [reply _posts/0000-01-06-skrishn.md:9]: 🐄

**Review by skrishn — COMMENTED** [review]
?
@jimmieego : please take look 🍺

**Pull request merged**